ext/date/php_date.stub.php

```diff
@@ -260,17 +260,17 @@ function date_default_timezone_get(): string {}
 
 /**
  * @refcount 1
- * @deprecated
  */
+#[\Deprecated]
 function date_sunrise(
     int $timestamp, int $returnFormat = SUNFUNCS_RET_STRING,
     ?float $latitude = null, ?float $longitude = null, ?float $zenith = null,
     ?float $utcOffset = null): string|int|float|false {}
 
 /**
  * @refcount 1
- * @deprecated
  */
+#[\Deprecated]
```

Comment:
Use date_sun_info() instead.
```suggestion
#[\Deprecated(message: 'Use date_sun_info() instead', since: '8.1')]
```